next-i18next.config.js/node.js

```diff
@@ -0,0 +1,25 @@
+const ChainedBackend = require('i18next-chained-backend');
+const FsBackend = require('i18next-fs-backend/cjs');
+const HttpBackend = require('i18next-http-backend/cjs');
+
+const config = require('./index');
+
+const { backend } = config;
+
+module.exports = {
+  ...config,
+
+  use: [ChainedBackend],
+
+  backend: {
+    backends: [FsBackend, HttpBackend],
```

Comment:
For the `next build`, we will priorities the locale files under the `packages/i18n` folder.